src/reducers/selectedApplicant.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Maintains the state of which applicant form is currently selected; used to determine which tab is displayed.
+ */
+
+// Simr is a Reducer-creator library that is very useful for creating getter/setter reducers that don't have
+// to maintain complicated state.  It makes creating these reducers very clean.
+import Simr from 'simr';
+
+export const selectedApplicantReducer = new Simr.Reducer('selectedApplicant', {count: 1, id: 0});
+selectedApplicantReducer.addSetter('id');
+selectedApplicantReducer.addSetter('count');
+
+// Action creator that sets the `selectedApplicant.selectedApplicant` value
+export const setSelectedApplicant = index => Simr.actions.setOn('selectedApplicant', 'id', index);
+
+export const setJointApplicantCount = count => Simr.actions.setOn('selectedApplicant', 'count', count);
```

Comment:
I'm all for shortcuts with reducers. I have played with so many different ways of simplifying reducers. Zach even has a library for it. https://github.com/zachcoyle/redux-looking-glass

I am up for anything. I just want to see a more functional, composable interface. My bias is against things that make actions for you, but I'm up for that. Ultimately, the decision is on you guys not me.